### ISSUE TITLE
Italic text is not recognized when "/" is present inside

### ISSUE THREAD

**Issue by outloudvi** [misskey-dev/mfm.js#102]
* Normal case: `*a*`

``` json
[{"type":"italic","children":[{"type":"text","props":{"text":"a"}}]}]
```

* Problematic case: `*a/n*`

``` json
[{"type":"text","props":{"text":"*a/n*"}}]
```

* Expected: `*a/n*`

``` json
[{"type":"italic","children":[{"type":"text","props":{"text":"a/n"}}]}]
```

**Comment by marihachi** [comment]
仕様通りの動作であり、バグではありません。
もし変更する場合は、MFM側の仕様変更が必要になります。

**Comment by outloudvi** [comment]
I think it would be okay if it is - as you said - per the MFM spec (please correct me if I had it wrong since I don't know much about Japanese). Also, I can [wrap italic text with `<i>`](https://wiki.misskey.io/ja/function/mfm#%E6%96%9C%E4%BD%93) which will not suffer from this problem.

I just feel it a little weird because it does not affect the bold notation:

``` jsonc
// *a/b*
[{"type":"text","props":{"text":"*a/b*"}}]
// **a/b**
[{"type":"bold","children":[{"type":"text","props":{"text":"a/b"}}]}]
```

**Comment by Johann150** [comment]
I agree that the current behaviour is very unexpected (and might thus be considered a bug). I personally consider the current behaviour for `**bold**` to be correct, so I think we should make the characters allowed inside `*italic*` the same as allowed in `<i>italic</i>`. (But `*italic*` should still require white space around it to be recognized.) Alternatively, we could remove the star-italic all together but that is not a good idea for backward compatibility.

(I did not quite understand; is the MFM specification somewhere else? Where would it have to be changed, since you said that "MFM itself has to be changed". Maybe that is just my translator not working correctly.)

**Comment by marihachi** [comment]
@Johann150 
以下にmfm.jsを実装するために作成された仕様があります。
https://github.com/misskey-dev/mfm.js/blob/develop/docs/syntax.md

**Comment by ThatOneCalculator** [comment]
Any update on this?

**Comment by outloudvi** [comment]
> Any update on this?

@ThatOneCalculator Per marihachi, it seems to be expected rather than a bug:

> 仕様通りの動作です。これはバグではありません。

A workaround is to use `<i>text</i>` as instructed by the MFM docs.

**Comment by ThatOneCalculator** [comment]
I personally disagree, as this isn't the behavior of other standard markdown parsers and confuses users.

**Comment by tassoman** [comment]
I join the discussion to say actual behaviour, I described into #134, it's confusing for people writing Latin / European languages.  

Although in this repository `MFM` is never addressed as **Markdown For Misskey** in the source code, or documentation, neither when the repository was renamed #30 (2021/03).

I want to note that [since 2021/09 the Misskey.io Wiki Pages are claiming MFM is Markdown For Misskey](https://web.archive.org/web/20210925012823/https://wiki.misskey.io/ja/function/mfm), and not a different thing, for example _Markup For Misskey_.

This is the origin of confusion. Since the beginning, people expect to have markdown into Misskey. At least a basic subset, made of **bold**, _oblique_, links and quote. (No support for bullet / numbered lists)

So I end up asking for _oblique_ text rendering to be fixed. Please.

**Comment by ThatOneCalculator** [comment]
The Firefish project will be addressing this in [libmfm](https://git.joinfirefish.org/firefish/libmfm)

**Comment by tassoman** [comment]
That software can't be used in this project because is lacking of a free software license compatible with AGPLv3.  
If your code incorporated misskey-dev/mfm.js code, it must be released at least under MIT license or kept private.

**Comment by ThatOneCalculator** [comment]
The idea is it to be an alternative implementation to mfm.js.
